### ISSUE TITLE
A11y Accessibility: After applying resize, "Editor content" textbox is not visible.

### ISSUE THREAD

**Issue by joe-braley** [konveyor/static-report#132]
Steps to Reproduce
  1. Install kantra tool and execute an assessment.
  2. Change system resolution to 1280 x 768.
  3. With the report open:
    a. TAB until "hamburger" icon and press ENTER key.
    b. TAB until "Issues" tab and press ENTER key.
    c. TAB until "Detail" key (`>`) in first column and press ENTER key.
    d. TAB until first file entry (e.g., pom.xml) and press ENTER key.
    e. TAB until "Editor content" text box.
    f. Apply resize to 200%
    g. Observe the "Editor content" text box is no longer visible.

Expected Behavior
After applying resize, "Editor content" textbox should be visible.

Actual Behavior
After applying resize, "Editor content" textbox is not visible.

Environment Details
	• Operating System: Windows 11 Enterprise
        - Software Version: Microsoft Edge Version 130.0.2849.68 (Official build) (64-bit)

Other Resources:
![Image](https://github.com/user-attachments/assets/dec19b24-9a5b-4445-9d71-84b0e951fc91)

**Comment by joe-braley** [comment]
CC @kthatipally
